client/src/test/java/org/spine3/base/MismatchShould.java

```diff
@@ -33,33 +36,70 @@
     private static final String EXPECTED = "expected";
     private static final String FOUND = "found";
     private static final int VERSION = 0;
+    public static final String DEFAULT_VALUE = "";
 
     @Test
     public void has_private_constructor() {
         assertTrue(hasPrivateUtilityConstructor(Mismatch.class));
     }
 
     @Test
-    public void return_value_with_default_expected_field_if_it_was_passed_as_null() {
+    public void return_default_expected_value_if_it_was_passed_as_null() {
         final ValueMismatch mismatch = Mismatch.of(null, FOUND, REQUESTED, VERSION);
         final String expected = mismatch.getExpected()
                                         .toString();
-        assertEquals("", expected);
+
+        assertEquals(DEFAULT_VALUE, expected);
     }
 
     @Test
-    public void return_value_with_default_found_field_if_it_was_passed_as_null() {
+    public void return_default_found_value_if_it_was_passed_as_null() {
         final ValueMismatch mismatch = Mismatch.of(EXPECTED, null, REQUESTED, VERSION);
         final String actual = mismatch.getActual()
                                       .toString();
-        assertEquals("", actual);
+
+        assertEquals(DEFAULT_VALUE, actual);
     }
 
     @Test
-    public void return_value_with_passed_values() {
+    public void return_mismatch_object_with_specified_values_string_overload() {
```

Comment:
Please rename to `return_mismatch_object_with_string_values`. And other test methods.